orchard/orchard_client.go

```diff
@@ -24,11 +24,12 @@ type OrchardClient interface {
 }
 
 type OrchardRestClient struct {
-	Host string
+	Host   string
+	Runner OrchardRunner
```

Comment:
is this still necessary?

Reply:
I think given that there is an OrchardRunner interface, it seems appropriate to let the client pick an implementation of the OrchardRunner interface.  However, we have a single implementation OrchardStdoutRunner, it could seem redundant here.  Do you want to take it out here?

Reply:
let's take it out and add when we add new Runner implementation

Reply:
taken out in new commit